examples/chip-tool/commands/common/ModelCommand.cpp

```diff
@@ -81,6 +81,13 @@ void ModelCommand::SendCommand(ChipDeviceController * dc)
     dc->SendMessage(NULL, buffer);
 }
 
+bool _isValidFrame(uint8_t frameControl)
```

Comment:
No leading `_`, please, but do put it in an anonymous namespace.

Reply:
Any reason to not use it a a private method of ModelCommand ?

Reply:
I prefer anonymous namespace as well as this is mostly a utility method that is not necessarily integral to the class and its responsibilities.